Linux/Car_Client/chronos.cpp

```diff
@@ -35,7 +35,8 @@ Chronos::Chronos(QObject *parent) : QObject(parent)
             this, SLOT(processOpro(chronos_opro)));
     connect(mChronos, SIGNAL(rcmmRx(chronos_rcmm)),
             this, SLOT(processRcmm(chronos_rcmm)));
-
+	connect(mChronos, SIGNAL(heabTimeOut()),
+			this, SLOT(Chronos::abort()) );
```

Comment:
Chronos:: can be removed, right?

Reply:
It should be fine. I wanted to make it extra clear that I am not calling the abort from stdlib though.
```
* Abort execution and generate a core-dump.  */
extern void abort (void) __THROW __attribute__ ((__noreturn__));`

```

Reply:
Ah, makes sense. Although, when I'm running my build on the pi I get this output:
... No such slot Chronos::Chronos::abort()

Reply:
Aha, hmm. It looks like it gets 2x namespaces? Maybe I was just too careful :D